ASTree.cpp

```diff
@@ -2556,6 +2556,94 @@ PycRef<ASTNode> BuildFromCode(PycRef<PycCode> code, PycModule* mod)
         case Pyc::RESUME_A:
             /* Treated as no-op for decompyle purposes */
             break;
+        case Pyc::PUSH_NULL:
+            {
+                stack.push(NULL);
+            }
+            break;
+        case Pyc::PRECALL_A:
+            /* Treated as no-op for decompyle purposes */
+            break;
+        case Pyc::CALL_A:
```

Comment:
If this is identical to (or almost identical to) the `Pyc::CALL_FUNCTION_A` case, they should be combined rather than copied.

Reply:
You can add a conditional on the opcode as has been done in a few other places within shared case branches...  For example, something like:

```c++
        ...
        stack.pop();
        if (opcode == Pyc::CALL_A)
            stack.pop();
        ...
```

Reply:
Yeah I thought about it but not exactly about a conditional yeah I will modify this right now on my side and push it